README.md

```diff
@@ -1,37 +1,110 @@
+
+While the contents of the corpus will change due to curation and expansion, we aim to keep the deliverable API, the `corpus/` folder, as stable as possible. This means we avoid relocating files or folders, changing formats, changing columns in metadata files, or any other changes that might break downstream scripts. Conversely, files outside the `corpus/` folder are internal to the project. End users may find utility in them but we make no effort to keep them consistent.
+
+The data in the corpus is delivered as TEI XML files to follow established practices. The metadata is delivered as CSV files, following a [normal form](https://en.wikipedia.org/wiki/Database_normalization) database structure while allowing for a legible git history. A more detailed description of the data and metadata structure and formats can be found in the README files in the `corpus/` folder.
+
+## Descriptive statistics at a glance
+
+Currently, we have an extensive set of Parliamentary Records (Riksdagens Protokoll) from 1867 until now. We are in the process of preparing Motions for inclusion in the corpus and other document types will follow.
+
+|                                      |   v0.10.0  |
+|--------------------------------------|------------|
+|           Corpus size (GB)           |    5.58    |
+|    Number of parliamentary records   |    17642   |
+|   Total parliamentary record pages*  |   1041807  |
+|  Total parliamentary record speeches |   1127027  |
+|   Total parliamentary record words   |  441525242 |
+|           Number of Motions          |      0     |
+|          Total motion pages          |      0     |
+|          Total motion words          |      0     |
+|     Number of people with MP role    |    5975    |
+|  Number of people with minister role |     535    |
+
+\* Digital original parliamentary records for some years in the 1990s are not paginated and thus do not contribute to the page count.See also §_Number of Pages in Parliamentary Records_.
+
+### Parliamentary Records over time
+
+#### Number of Parliamentary Records
+
+![Number of Parliamentary Records](scripts/stats-dashboard/figures/n-prot/n-prot.png)
+
+#### Number of Pages in Parliamentary Records
+
+![Number of Pages in Parliamentary Records](scripts/stats-dashboard/figures/prot-pages/prot-pages.png)
+
+#### Number of Speeches in Parliamentary Records
+
+![Number of Speeches in Parliamentary Records](scripts/stats-dashboard/figures/prot-speeches/prot-speeches.png)
+
+Note: We are aware of an issue whereby speeches are over counted in the data's current form in the years after 2014 -- we're working on a fix. Until then, the following static graph is a better representation of the actual speeches in the Parliamentary Records for those years.
```

Comment:
This is fixed now, so maybe remove?

Reply:
it's not fixed until version {current}+1, but indeed, when we get stats that look more like Tomas', I will remove this

Reply:
Ok!